doc/playbook.rst

```diff
@@ -65,6 +65,29 @@ Any subsequent builds of the service container will build against both bases.
 
 This is the only situation in which a change to ``BASE_TAG_LIST`` should be committed to ``main``.
 
+Release Procedure
+-----------------
+Releases are largely automated through GitHub Actions (see the `ci-release.yaml <https://github.com/lsst-dm/prompt_processing/actions/workflows/ci-release.yaml>`  workflow file for details). When a semantic version tag is pushed to GitHub, Prompt Processing Docker images are published on GitHub and Docker Hub with that version.
+
+Regular releases happen from the ``main`` branch after changes have been merged.
+From the ``main`` branch you can release a new major version (``X.0.0``), a new minor version of the current major version (``X.Y.0``), or a new patch of the current major-minor version (``X.Y.Z``).  Release tags are semantic version identifiers following the :pep:`440` specification.
+
+1. Create a Release
+
+On GitHub.com, navigate to the main page of the repository.  To the right of the list of files, click **Releases**.  At the top of the page, click **Draft a new release**.  Type a tag using semantic versioning described in the previous section.  The Target should be the main branch.  
+
+Then select **Generate Release Notes**.  Review and make any changes then select **Publish Release**.
```

Comment:
We might want to require some standard information in the release description (possibly more a question for @ktlim). For my part, I'd want to record the underlying Science Pipelines version (especially if it can't be stored in the container as a label!), because that's something that _everybody_ will be asking about Prompt Processing.

Reply:
We discussed this at PP standup and came up with the following list:
- Any specific motivations for the release (for example, including a specific feature, fix, or compatibility change)
- Science Pipelines version and rubin-env version
- Changes to the APDB and Alerts schemas, to be replaced by version numbers once we have them.

I can go ahead and add the version numbers to the 1.0.0 description, since I can probably get those more easily than @dspeck1.

Reply:
Added the notes above on release notes into the playbook.